packages/vc-verification/test/issuer-verification-test.ts

```diff
@@ -627,6 +627,78 @@ function testSuite() {
       ).to.be.true;
     });
 
+    it('RoleEIP191Jwt without claimType should not break verification,', async () => {
+      const adminJWT = new JWT(adminKeys);
+      const claimAdmin = {
+        claimData: { fields: {}, claimTypeVersion: 1 },
+        iss: adminDid,
+        credentialStatus: {
+          id: 'https://identitycache-dev.energyweb.org/v1/status-list/urn:uuid:4fb4e120-a566-499c-85fb-47bb5abd3d6b',
+          type: 'StatusList2021Entry',
+          statusPurpose: 'revocation',
+          statusListIndex: '0',
+          statusListCredential:
+            'https://identitycache-dev.energyweb.org/v1/status-list/urn:uuid:4fb4e120-a566-499c-85fb-47bb5abd3d6b',
+        },
+        signer: adminDid,
+      };
+      const token = await adminJWT.sign(claimAdmin);
+      const ipfsCIDAdmin = await didStore.save(token);
+      const serviceIdAdmin = adminRole;
+      const updateDataAdmin: IUpdateData = {
+        type: DIDAttribute.ServicePoint,
+        value: {
+          id: `${adminDid}#service-${serviceIdAdmin}`,
+          type: 'ClaimStore',
+          serviceEndpoint: ipfsCIDAdmin,
+        },
+      };
+      await adminOperator.update(
+        adminDid,
+        DIDAttribute.ServicePoint,
+        updateDataAdmin,
+        validity
+      );
+
+      const managerRoleClaim = {
+        claimData: { fields: {}, claimTypeVersion: 1, claimType: managerRole },
+        iss: adminDid,
+        credentialStatus: {
+          id: 'https://identitycache-dev.energyweb.org/v1/status-list/urn:uuid:4fb4e120-a566-499c-85fb-47bb5abd3d6b',
+          type: 'StatusList2021Entry',
+          statusPurpose: 'revocation',
+          statusListIndex: '0',
+          statusListCredential:
+            'https://identitycache-dev.energyweb.org/v1/status-list/urn:uuid:4fb4e120-a566-499c-85fb-47bb5abd3d6b',
+        },
+        signer: adminDid,
+      };
+      const tokenManager = await adminJWT.sign(managerRoleClaim);
+      const ipfsCIDManager = await didStore.save(tokenManager);
+      const serviceIdManager = managerRole;
+      const updateDataManager: IUpdateData = {
+        type: DIDAttribute.ServicePoint,
+        value: {
+          id: `${managerDid}#service-${serviceIdManager}`,
+          type: 'ClaimStore',
+          serviceEndpoint: ipfsCIDManager,
+        },
+      };
```

Comment:
Would be good to apply the same naming convention here as well IMO. I think it is more readable. Having the adjective first is more intuitive, IMO.
```suggestion
      const managerToken = await adminJWT.sign(managerRoleClaim);
      const managerTokenCID = await didStore.save(tokenManager);
      const managerServiceId = managerRole;
      const managerUpdateData: IUpdateData = {
        type: DIDAttribute.ServicePoint,
        value: {
          id: `${managerDid}#service-${serviceIdManager}`,
          type: 'ClaimStore',
          serviceEndpoint: ipfsCIDManager,
        },
      };
```